package.json

```
@@ -31,9 +31,10 @@
   "scripts": {
     "prebuild": "rimraf ./dist",
     "build:esm:package": "echo '{ \"type\": \"module\" }' > ./dist/esm/package.json",
-    "build:esm": "swc ./src -d ./dist/esm --ignore **/*.d.ts --strip-leading-paths -C module.type=es6 -C module.ignoreDynamic=true",
-    "build:cjs": "swc ./src -d ./dist/cjs --ignore **/*.d.ts --strip-leading-paths -C module.type=commonjs -C module.ignoreDynamic=true",
-    "build": "npm run build:cjs && npm run build:esm && npm run build:esm:package",
+    "build:esm": "npx esbuild ./src/index.ts  --bundle --format=esm --tree-shaking=true --minify --outdir=./dist/esm --format=esm --platform=node --define:BROWSER_BUILD=false --external:electron",
+    "build:cjs": "npx esbuild ./src/index.ts  --bundle --format=cjs --tree-shaking=true --minify --outdir=./dist/cjs --format=cjs --platform=node --define:BROWSER_BUILD=false --external:electron",
+    "build:browser": "npx esbuild ./src/index.ts --bundle --format=esm --tree-shaking=true --minify --outdir=./dist/browser --platform=browser --define:BROWSER_BUILD=true --external:electron",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:browser && npm run build:esm:package",
```

Looks more logical to me to keep ESM-related scripts together in the chain...

```suggestion
    "build": "npm run build:cjs && npm run build:esm && npm run build:esm:package && npm run build:browser",
```